### PR TITLE
Group tasks

### LINKED ISSUE #22: [1pt] Create empty My profile - Setup (group task)
- Create a route and a view component. Use `<NavLink />` for the page navigation links and style active class to indicate which section/page the user is currently on (underline active navigation link).
- This view should be empty - you will add content in separate tasks.

### LINKED ISSUE #23: [1pt] Create website header with empty nav - Setup (group task)
- Download the [free image](https://www.flaticon.com/free-icon/planet_3212567?term=space&page=1&position=19&page=1&position=19&related_id=3212567&origin=style) for the app logo.
- Create **empty** navigation (this will be filled in in separate tasks).

### LINKED ISSUE #24: [1pt] Create app and install basic dependencies - Setup (group task)
- Create React App.
- Install [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).

### PR DESCRIPTION
**Pull request by victormc13**
# Latest Tasks Completed

We completed the following tasks:
  - https://github.com/victormc13/Space-Travelers-Hub/projects/1#card-90340084
  - https://github.com/victormc13/Space-Travelers-Hub/projects/1#card-90340083
  - https://github.com/victormc13/Space-Travelers-Hub/projects/1#card-90340079

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,261 @@
...
+- **[Use of React to the functionalities of the page]**
 
-To learn React, check out the [React documentation](https://reactjs.org/).
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-### Code Splitting
+<!-- LIVE DEMO -->
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+<!-- ## 🚀 Live Demo <a name="live-demo"></a> -->
```
Comment (obibaadoma) [inline README.md:84]: Link to the live demo can be taken out since the project is not hosted

**Review by ahmidou09 — APPROVED** [review]
Hi team,

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉

Highlights
📌 No Linters error ✅
✅ Great use of Git flow. 💯
a

File: MIT.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+## Copyright 2023, Victor Miñope
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this [APP TYPE] and associated documentation files, to deal in the [APP TYPE] without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the [APP TYPE], and to permit persons to whom the [APP TYPE] is furnished to do so, subject to the following conditions:
```
Comment (obibaadoma) [inline MIT.md:3]: [Optional] Kindly include the name of the project for which is licence is being applied to

**Review by obibaadoma — APPROVED** [review]
I have nothing much to say than it time to merge the code. Great work and keep it up.

**Pull request merged**